const hunterGames = async () => {
    try {

    } catch (err) {
        throw err;
    }
}